### DISCUSSION TITLE
How 2FA works

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [near/near-wallet#2812] [Q&A] [answered]
Documentation here https://nearhelp.zendesk.com/hc/en-us/articles/360062542533-Two-Factor-Authentication-2FA- says that some contract is deployed, to guard your account, but where code of that contract can be found?

**Comment by roshaans** [comment, marked as answer]
You may find the code in [`near/core-contracts/multisig`](https://github.com/near/core-contracts/blob/master/multisig/src/lib.rs)
